composer.json

```diff
@@ -56,13 +56,13 @@
         "utopia-php/image": "0.5.*",
         "utopia-php/locale": "0.4.*",
         "utopia-php/logger": "0.3.*",
-        "utopia-php/messaging": "0.1.*",
+        "utopia-php/messaging": "0.2.*",
         "utopia-php/migration": "0.3.*",
         "utopia-php/orchestration": "0.9.*",
-        "utopia-php/platform": "0.4.*",
+        "utopia-php/platform": "dev-integrate-workers as 0.3.3",
         "utopia-php/pools": "0.4.*",
         "utopia-php/preloader": "0.2.*",
-        "utopia-php/queue": "0.5.*",
+        "utopia-php/queue": "dev-feat-get-worker-start as 0.5.3",
         "utopia-php/registry": "0.5.*",
```

Comment:
We need to get the proper library versions released before merging to 1.4.x

Reply:
Cool

Reply:
fixed